site/content/docs/03-client-api.md

```diff
@@ -24,9 +24,10 @@ sapper.start({
 ```
 
 
-### goto(href, options?)
+### goto(href, state?, options?)
```

Comment:
it looks like the docs need to be updated to match the recent changes

Reply:
Should be ok now, i've merged with new sapper version.